### PR TITLE
Vue sign in page new style

### PR DESCRIPTION
**Pull request by ErikCH**
*Issue #, if available:* [Asana](https://app.asana.com/0/0/1200953035425696/f)

*Description of changes:*
Added new styles to Vue

image

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: examples/vue/src/main.ts [hunk 1]
```diff
@@ -1,7 +1,6 @@
-import { createApp } from "vue";
-import App from "./App.vue";
-import router from "./router";
+import { createApp } from 'vue';
+import App from './App.vue';
+import router from './router';
+import './styles.scss';
```
Comment (ErikCH) [inline examples/vue/src/main.ts:4]: There are styles in the next example, that I copied over to the Vue example so the look and feel would match.

File: packages/vue/src/components/sign-in-password-control.vue [hunk 2]
```diff
@@ -1,19 +1,82 @@
...
+        class="
+          amplify-button
+          amplify-field-group__control
+          amplify-field__show-password
+        "
+        data-fullwidth="false"
+        type="button"
+        @click="togglePasswordText"
+      >
+        <svg
```
Comment (ErikCH) [inline packages/vue/src/components/sign-in-password-control.vue:31]: I inlined the SVGS for now.

**Review by eddiekeller — APPROVED** [review]

File: examples/vue/src/main.ts (hunk 1 above)
Comment (wlee221) [reply examples/vue/src/main.ts:4]: Gotcha, I'll probably have to do this in Angular too... and hope Angular is cooperative with scss.

Comment (wlee221) [reply examples/vue/src/main.ts:4]: Do you recall which primitives needed this scss file? I suppose this is filling in the gap of not having primitives on Angular/Vue.

**Review by wlee221 — APPROVED** [review]
✨  LGTM! I will for sure reference this when I get to Angular parity styles too 😛

File: examples/vue/src/main.ts (hunk 1 above)
Comment (ErikCH) [reply examples/vue/src/main.ts:4]: All the ones with the !import were used for sure. Also I added a font.

**Pull request merged**